app/views/practices/_adoptions_list.html.erb

```diff
@@ -18,6 +18,9 @@
               </span>
       <span class="line-height-sans-505 display-inline-block facility-name-width vertical-align-top">
                 <%= facility["OfficialStationName"] %>
+                <% unless facility["OfficialStationName"].downcase.include?(facility["CommonName"].downcase) %>
```

Comment:
is there a way we can put this into a view helper file?
```
<% unless facility["OfficialStationName"].include?(facility["CommonName"]) %>
      (<%= facility["CommonName"] %>)
<% end %>
```

Reply:
ooo good idea~ will do :)